### PR TITLE
Add Goldman Sachs

### LINKED ISSUE #9777: Add Goldman Sachs
### Brand Name

Goldman Sachs

### Website

https://www.goldmansachs.com/

### Popularity Metric

https://www.similarweb.com/website/goldmansachs.com/

### Official Resources for Icon and Color

https://design.gs.com/brand/goldman-sachs-logo

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by imsherlocked**
**Issue:** closes #9777

**Similarweb rank:**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by imsherlocked** [comment]
Hello @adamrusted 
I am having some issues for the checks, can you please help me out for this,

**Comment by PeterShaggyNoble** [comment]
@imsherlocked, that's being caused by the newline you have after the `source` in the JSON file.

Could you also please update this PR to provide all information requested in the template.

**Comment by imsherlocked** [comment]
@PeterShaggyNoble 
I have made the changes in the json file but its still giving issue in the checks.
I have attached the snip for the json file below.
Issue

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5299,10 +5299,16 @@
...
+            "title": "Goldman Sachs",
+            "hex": "000000",
+            "source": "https://www.goldmansachs.com"
+        },
         {
             "title": "Goodreads",
             "hex": "372213",
             "source": "https://www.goodreads.com/about/press"
+            
         },
```
Comment (adamrusted) [inline _data/simple-icons.json:5312]:
```suggestion
            "source": "https://www.goodreads.com/about/press"
        },
```

File: icons/goldmansachs.svg [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
... 1.65v.82h-9.9v-.82l1.1-1.65V15.48l-1.1-1.32v-.82l7.92-.94h.66l.22 3.08c.93-2.42 3.24-3.74 6.38-3.74 3.35 0 5.55 1.65 6.54 4.62.82-2.86 3.41-4.62 6.71-4.62 4.34 0 7.09 2.86 7.09 8.41zm-10.28 36.41c0 4.78-3.08 8.74-10.5 8.74-1.92 0-4.51-.22-8.52-.94V71.85h2.25l.71 2.2c1.76 5.61 3.46 7.7 6.16 7.7 2.53 0 4.18-1.82 4.18-4.62 0-2.31-1.16-3.9-3.9-5.39l-4.73-2.53c-3.02-1.65-4.67-4.23-4.67-7.42 0-5 3.91-8.03 10.39-8.03 1.87 0 4.29.27 6.93.82v8.03h-2.25l-1.16-2.92c-1.32-3.52-2.58-4.67-4.67-4.67-2.25 0-3.57 1.32-3.57 3.52 0 1.87 1.1 3.13 3.79 4.67l4.67 2.64c3.18 1.82 4.89 4.4 4.89 8.42zm35.58-36.41l1.1 1.65v.82h-8.41l-.38-3.19c-.88 2.42-3.19 3.85-6.16 3.85-4.4 0-7.15-2.75-7.15-7.26 0-4.4 2.47-7.31 10.06-10.28l3.24-1.26v-4.12c0-3.63-.66-5.11-2.69-5.11-1.65 0-2.58.88-4.18 4.89l-1.81 4.51h-2.48V13.4c3.25-.99 6.49-1.65 9.29-1.65 6.21 0 9.57 2.97 9.57 8.36zm-7.7-4.62v-9.51l-1.04.44c-3.24 1.32-4.45 4.29-4.45 8.63 0 3.25.77 5.12 2.58 5.12s2.91-1.76 2.91-4.68zm35.96 6.27v.82h-9.9v-.82l1.1-1.65v-19.8c0-2.37-.71-3.79-2.42-3.79-2.04 0-3.52 1.71-3.52 5.06v18.53l1.1 1.65v.82h-9.9v-.82l1.1-1.65V15.48l-1.1-1.32v-.82l7.92-.94h.66l.22 3.08c.93-2.42 3.19-3.74 6.43-3.74 4.51 0 7.2 2.86 7.2 8.41v17.71z"/></svg>
```
Comment (adamrusted) [inline icons/goldmansachs.svg:2]:
```suggestion
<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" width="2500" height="1051"><path d="M16.92 21.2H28.3v18.64c-3.8.77-6.71 1.16-9.9 1.16C5.92 41 1.52 33.85 1.52 22.08c0-12.65 6.1-19.8 18.26-19.8 1.54 0 4.45.11 8.08.66v11.93h-2.64l-1.49-4.56C22.3 5.97 20.6 3.82 17.9 3.82c-4.67 0-7.15 6.32-7.15 18.15 0 11.88.99 17.49 7.09 17.49.83 0 1.81-.11 2.75-.38V25.21l-3.68-2.53V21.2zm11.16 50.04c0 7.48-5.11 11.77-14.13 11.77-2.47 0-6.1-.38-9.51-.94V67.45h2.64l1.1 3.46c2.47 7.97 4.23 10.61 7.37 10.61 3.3 0 5.66-2.91 5.66-6.76 0-3.13-1.48-5.33-4.67-7.04l-5.5-2.97C6.8 62.5 4.49 58.98 4.49 54.69c0-6.49 5.33-10.39 14.08-10.39 1.98 0 4.29.17 7.97.66v11.93H23.9l-1.48-4.56c-1.49-4.84-3.08-6.6-5.99-6.6-3.25 0-5.17 2.09-5.17 5.11 0 2.69 1.49 4.62 4.4 6.27l5.55 3.13c4.67 2.64 6.87 6.16 6.87 11zm24.31 8.64l1.1 1.65v.83h-8.41l-.39-3.19c-.88 2.42-3.19 3.85-6.16 3.85-4.4 0-7.15-2.75-7.15-7.26 0-4.4 2.47-7.31 10.06-10.28l3.24-1.26V60.1c0-3.63-.66-5.11-2.69-5.11-1.65 0-2.58.88-4.18 4.89L36 64.37h-2.47V55.4c3.24-.99 6.49-1.65 9.29-1.65 6.21 0 9.57 2.97 9.57 8.36zm3.18-53.51c0 9.35-5 14.63-11.44 14.63-6.43 0-11.43-5.28-11.43-14.63s5-14.63 11.44-14.63c6.43 0 11.43 5.28 11.43 14.63zM44.69 75.26v-9.51l-1.04.44c-3.25 1.32-4.46 4.29-4.46 8.63 0 3.25.77 5.11 2.58 5.11s2.92-1.76 2.92-4.67zm2.8-48.89c0-10.56-.88-13.36-3.35-13.36s-3.35 2.8-3.35 13.36.88 13.36 3.35 13.36 3.35-2.8 3.35-13.36zM75.1 75.53l.99.5c-1.76 5.06-4.01 6.98-8.36 6.98-7.09 0-11.05-5.33-11.05-14.63 0-9.29 4.62-14.63 12.15-14.63 2.03 0 4.73.33 7.04 1.26v9.35h-2.58l-1.04-3.52c-1.38-4.67-2.14-5.88-3.52-5.88-2.2 0-3.96 2.86-3.96 13.75 0 8.74 1.32 11.93 4.67 11.93 2.25.01 4.17-1.42 5.66-5.11zm-6.11-35.19h-9.9v-.82l1.1-1.65V7.18l-1.1-1.65V4.7l8.8-.66v33.82l1.1 1.65zm13.53.66c-6.05 0-9.68-5.5-9.68-14.9 0-9.35 3.79-14.35 9.62-14.35 2.97 0 4.62 1.15 5.33 3.46V7.18l-1.1-1.65V4.7l8.8-.66v33.82l1.1 1.65v.82h-7.81l-.6-4.18c-.87 3.58-2.52 4.85-5.66 4.85zm19.3 38.88l1.1 1.65v.83h-9.9v-.83l1.1-1.65v-19.8c0-2.36-.71-3.79-2.42-3.79-2.04 0-3.52 1.71-3.52 5.06v18.53l1.1 1.65v.83h-9.9v-.83l1.1-1.65V49.19l-1.1-1.65v-.82l8.8-.66v11.38c.99-2.36 3.24-3.68 6.43-3.68 4.51 0 7.2 2.86 7.2 8.41v17.71zM80.93 26.2c0 9.9.88 12.92 3.3 12.92 3.02 0 3.57-4.34 3.57-13.03 0-8.41-.6-12.59-3.52-12.59-2.47 0-3.35 2.97-3.35 12.7zm55.21 11.66l1.1 1.65v.82h-9.9v-.82l1.1-1.65v-20.4c0-1.98-.66-3.19-2.2-3.19-1.92 0-3.46 1.71-3.46 5v18.59l1.1 1.65v.82h-9.9v-.82l1.1-1.65v-20.4c0-1.98-.66-3.19-2.2-3.19-1.92 0-3.46 1.71-3.46 5v18.59l1.1 1.65v.82h-9.9v-.82l1.1-1.65V15.48l-1.1-1.32v-.82l7.92-.94h.66l.22 3.08c.93-2.42 3.24-3.74 6.38-3.74 3.35 0 5.55 1.65 6.54 4.62.82-2.86 3.41-4.62 6.71-4.62 4.34 0 7.09 2.86 7.09 8.41zm-10.28 36.41c0 4.78-3.08 8.74-10.5 8.74-1.92 0-4.51-.22-8.52-.94V71.85h2.25l.71 2.2c1.76 5.61 3.46 7.7 6.16 7.7 2.53 0 4.18-1.82 4.18-4.62 0-2.31-1.16-3.9-3.9-5.39l-4.73-2.53c-3.02-1.65-4.67-4.23-4.67-7.42 0-5 3.91-8.03 10.39-8.03 1.87 0 4.29.27 6.93.82v8.03h-2.25l-1.16-2.92c-1.32-3.52-2.58-4.67-4.67-4.67-2.25 0-3.57 1.32-3.57 3.52 0 1.87 1.1 3.13 3.79 4.67l4.67 2.64c3.18 1.82 4.89 4.4 4.89 8.42zm35.58-36.41l1.1 1.65v.82h-8.41l-.38-3.19c-.88 2.42-3.19 3.85-6.16 3.85-4.4 0-7.15-2.75-7.15-7.26 0-4.4 2.47-7.31 10.06-10.28l3.24-1.26v-4.12c0-3.63-.66-5.11-2.69-5.11-1.65 0-2.58.88-4.18 4.89l-1.81 4.51h-2.48V13.4c3.25-.99 6.49-1.65 9.29-1.65 6.21 0 9.57 2.97 9.57 8.36zm-7.7-4.62v-9.51l-1.04.44c-3.24 1.32-4.45 4.29-4.45 8.63 0 3.25.77 5.12 2.58 5.12s2.91-1.76 2.91-4.68zm35.96 6.27v.82h-9.9v-.82l1.1-1.65v-19.8c0-2.37-.71-3.79-2.42-3.79-2.04 0-3.52 1.71-3.52 5.06v18.53l1.1 1.65v.82h-9.9v-.82l1.1-1.65V15.48l-1.1-1.32v-.82l7.92-.94h.66l.22 3.08c.93-2.42 3.19-3.74 6.43-3.74 4.51 0 7.2 2.86 7.2 8.41v17.71z"/></svg>
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Hey @imsherlocked - the below suggestions _should_ resolve the linter issue with the JSON that you're having. The icon itself will need some more work (`npx svgo icons/goldmansachs.svg` is your friend!) but we can get to that after these fixes!

**Comment by imsherlocked** [comment]
Hello @adamrusted @PeterShaggyNoble 
Thank you for all the help. I was able to resolve one issue.
I am still unable to understand what issue the SVG has, the viewbox is 0 0 24 24 according to the guidelines but its still failing the check.
Can you please provide some guidance so that I can resolve the issue.

**Comment by adamrusted** [comment]
Hey @imsherlocked - it looks like the viewport is set correctly, but the path is now clipping outside of it. If you open the SVG in a vector imaging tool like Inkscape, you should be able to scale the image to fit the viewbox (keeping the aspect ratio, so as to not distort the logo). If you do that - and center the path, it _should_ pass linters once you've ran it through SVGO and done all those steps :)

**Comment by imsherlocked** [comment]
Hello @adamrusted 
I have been trying to fit the logo inside the viewbox by using inkscape and boxy but when I do so and then use SVG viewer to, it still clips outside.
Can you provide any guidance so that I can pass the check.
Thank you!!

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
The SVG should contain a `<title>` element and a single `path` element only; have a look at some of our existing files to see how they're structured.

File: icons/goldmansachs.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
....5,8.74 -1.92,0 -4.51,-0.22 -8.52,-0.94 V 71.85 h 2.25 l 0.71,2.2 c 1.76,5.61 3.46,7.7 6.16,7.7 2.53,0 4.18,-1.82 4.18,-4.62 0,-2.31 -1.16,-3.9 -3.9,-5.39 l -4.73,-2.53 c -3.02,-1.65 -4.67,-4.23 -4.67,-7.42 0,-5 3.91,-8.03 10.39,-8.03 1.87,0 4.29,0.27 6.93,0.82 v 8.03 h -2.25 l -1.16,-2.92 c -1.32,-3.52 -2.58,-4.67 -4.67,-4.67 -2.25,0 -3.57,1.32 -3.57,3.52 0,1.87 1.1,3.13 3.79,4.67 l 4.67,2.64 c 3.18,1.82 4.89,4.4 4.89,8.42 z m 35.58,-36.41 1.1,1.65 v 0.82 h -8.41 l -0.38,-3.19 c -0.88,2.42 -3.19,3.85 -6.16,3.85 -4.4,0 -7.15,-2.75 -7.15,-7.26 0,-4.4 2.47,-7.31 10.06,-10.28 l 3.24,-1.26 v -4.12 c 0,-3.63 -0.66,-5.11 -2.69,-5.11 -1.65,0 -2.58,0.88 -4.18,4.89 l -1.81,4.51 h -2.48 V 13.4 c 3.25,-0.99 6.49,-1.65 9.29,-1.65 6.21,0 9.57,2.97 9.57,8.36 z m -7.7,-4.62 v -9.51 l -1.04,0.44 c -3.24,1.32 -4.45,4.29 -4.45,8.63 0,3.25 0.77,5.12 2.58,5.12 1.81,0 2.91,-1.76 2.91,-4.68 z m 35.96,6.27 v 0.82 h -9.9 v -0.82 l 1.1,-1.65 v -19.8 c 0,-2.37 -0.71,-3.79 -2.42,-3.79 -2.04,0 -3.52,1.71 -3.52,5.06 v 18.53 l 1.1,1.65 v 0.82 h -9.9 v -0.82 l 1.1,-1.65 V 15.48 l -1.1,-1.32 v -0.82 l 7.92,-0.94 h 0.66 l 0.22,3.08 c 0.93,-2.42 3.19,-3.74 6.43,-3.74 4.51,0 7.2,2.86 7.2,8.41 v 17.71 z" /></g></svg>
```
Comment (adamrusted) [inline icons/goldmansachs.svg:7]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Goldman Sachs</title><path d="M1.963 9.264h1.452v2.378c-.484.098-.855.148-1.262.148C.561 11.79 0 10.878 0 9.377c0-1.614.778-2.526 2.329-2.526.196 0 .567.014 1.03.084v1.522h-.336l-.19-.582c-.183-.553-.4-.827-.744-.827-.596 0-.912.806-.912 2.315 0 1.515.126 2.23.904 2.23.106 0 .231-.014.351-.048V9.776l-.469-.323v-.189Zm1.424 6.382c0 .954-.651 1.501-1.802 1.501-.315 0-.778-.048-1.213-.119v-1.865h.337l.14.441c.315 1.017.54 1.353.94 1.353.421 0 .722-.371.722-.862 0-.399-.189-.68-.595-.898l-.702-.378c-.541-.287-.835-.736-.835-1.283 0-.828.68-1.326 1.796-1.326.252 0 .547.022 1.016.085v1.521h-.337l-.188-.581c-.19-.618-.393-.842-.764-.842-.415 0-.66.266-.66.652 0 .343.19.589.561.799l.708.399c.596.337.876.786.876 1.403Zm3.101 1.102.14.211v.106H5.556l-.05-.407c-.112.308-.407.491-.786.491-.561 0-.912-.351-.912-.926 0-.561.315-.933 1.283-1.311l.414-.161v-.525c0-.463-.085-.652-.344-.652-.21 0-.329.112-.533.623l-.231.573h-.315v-1.144c.414-.126.828-.21 1.185-.21.792 0 1.221.378 1.221 1.066v2.266Zm.405-6.824c0 1.192-.637 1.866-1.459 1.866-.82 0-1.457-.674-1.457-1.866 0-1.193.637-1.866 1.459-1.866.82 0 1.457.673 1.457 1.866Zm-1.387 6.235v-1.213l-.133.056c-.414.169-.569.547-.569 1.101 0 .414.099.652.329.652.231 0 .373-.225.373-.596Zm.357-6.235c0-1.347-.112-1.704-.427-1.704s-.428.357-.428 1.704c0 1.346.113 1.704.428 1.704s.427-.358.427-1.704Zm3.521 6.269.126.064c-.224.646-.511.89-1.066.89-.904 0-1.409-.679-1.409-1.865 0-1.185.589-1.866 1.55-1.866.258 0 .603.042.897.16v1.193h-.329l-.132-.449c-.176-.596-.273-.75-.449-.75-.281 0-.505.365-.505 1.754 0 1.114.168 1.521.595 1.521.287.001.532-.181.722-.652Zm-.779-4.488H7.342v-.104l.141-.211V7.476l-.141-.21V7.16l1.123-.084v4.313l.14.211v.105Zm1.726.085c-.772 0-1.235-.702-1.235-1.901 0-1.192.483-1.83 1.227-1.83.379 0 .589.147.68.441V7.476l-.141-.21V7.16l1.123-.084v4.313l.14.211v.104h-.996l-.077-.533c-.111.457-.321.619-.721.619Zm2.461 4.958.14.211v.106H11.67v-.106l.14-.211v-2.525c0-.301-.091-.483-.309-.483-.26 0-.449.218-.449.645v2.363l.141.211v.106H9.93v-.106l.14-.211v-3.914l-.14-.21v-.105l1.122-.084v1.451c.127-.301.414-.469.82-.469.576 0 .919.365.919 1.073v2.258h.001Zm-2.664-6.846c0 1.263.112 1.648.421 1.648.385 0 .455-.554.455-1.662 0-1.073-.077-1.606-.449-1.606-.315 0-.427.379-.427 1.62Zm7.041 1.487.14.211v.104h-1.262V11.6l.14-.211V8.787c0-.252-.084-.407-.281-.407-.244 0-.441.219-.441.638v2.371l.14.211v.104h-1.262V11.6l.14-.211V8.787c0-.252-.084-.407-.28-.407-.245 0-.442.219-.442.638v2.371l.141.211v.104h-1.263V11.6l.14-.211V8.535l-.14-.169v-.104l1.01-.12h.084l.028.393c.119-.309.414-.477.814-.477.427 0 .708.21.834.589.105-.365.435-.589.856-.589.553 0 .904.365.904 1.072v2.259Zm-1.311 4.644c0 .609-.393 1.114-1.339 1.114-.245 0-.575-.028-1.087-.119v-1.304h.287l.091.281c.224.715.441.982.785.982.323 0 .533-.232.533-.59 0-.294-.147-.497-.497-.687l-.603-.323c-.385-.21-.596-.539-.596-.946 0-.638.499-1.024 1.325-1.024.239 0 .547.034.884.105v1.024h-.287l-.148-.373c-.168-.449-.329-.595-.595-.595-.287 0-.456.168-.456.449 0 .238.141.399.484.595l.595.337c.406.232.624.561.624 1.074Zm4.538-4.644.14.211v.104h-1.073l-.048-.407c-.112.309-.407.491-.786.491-.561 0-.911-.35-.911-.926 0-.561.315-.932 1.283-1.311l.413-.16v-.526c0-.463-.084-.652-.343-.652-.211 0-.329.113-.533.624l-.231.575h-.317V8.27c.415-.127.828-.211 1.185-.211.792 0 1.221.379 1.221 1.066v2.264Zm-.982-.589V9.587l-.133.056c-.413.168-.567.547-.567 1.101 0 .414.098.653.329.653.23 0 .371-.225.371-.597Zm4.586.8v.104h-1.263V11.6l.141-.211V8.864c0-.302-.091-.484-.309-.484-.26 0-.449.219-.449.646v2.363l.14.211v.104h-1.262V11.6l.14-.211V8.535l-.14-.169v-.104l1.01-.12h.084l.028.393c.119-.309.407-.477.82-.477.575 0 .918.365.918 1.072v2.259L24 11.6Z"/></svg>
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Looks like the icon you're trying to submit is squashed to fit in the viewbox. I've generated (and confirmed there's no crazy errors) a path linked in the suggestions. You're welcome to use it - but we'll need another maintainer to review.

**Review by PeterShaggyNoble — APPROVED** [review]
Goldman Sachs have [a site for their brand](https://design.gs.com) and @adamrusted's path looks good when compared with [this SVG](https://design.gs.com/downloads/Goldman_Sachs_Signature.svg) found on [this page](https://design.gs.com/brand/goldman-sachs-logo). I'll update the JSON to reflect the findings on that site and then this can be merged in.

**Pull request merged**